### DISCUSSION TITLE
Happy Holidays 2023

### DISCUSSION THREAD

**Discussion by mergen3107** [koreader/koreader#11256] [General]
Happy Holidays, Merry Christmas, Happy New Year, Happy Hanukkah to whom it may concern!

Thank you very much, KOReader Community & Co., for making such a great reading experience and, most importantly, great hub where people can ask and raise their concerns.

KOReader is a true gateway to literature, of all kinds. 

Thank you! I wish you all and your families great health, prosperity and lots of happy moments in the upcoming year(s)!

**Comment by Frenzie** [comment]
Happy holidays!

**Reply by NiLuJe** [reply]
Ditto, happy holidays everyone ;).

**Reply by Galunid** [reply]
Likewise, happy holidays!

**Comment by mergen3107** [comment]
For fun statistics, in 2023 there have been 657+ successful commits (so far): https://github.com/koreader/koreader/compare/v2022.11...master

2022 had 663 commits: https://github.com/koreader/koreader/compare/v2021.12.1...v2022.11
2021 had 909 commits: https://github.com/koreader/koreader/compare/v2020.12...v2021.12.1
2020 had 664 commits: https://github.com/koreader/koreader/compare/v2019.12...v2020.12
2019 had 885 commits: https://github.com/koreader/koreader/compare/v2018.12...v2019.12
2018 had 885 commits: https://github.com/koreader/koreader/compare/v2017.10.04-nightly...v2018.12
2017 had 586 commits: https://github.com/koreader/koreader/compare/v2016.11.20-nightly...v2017.10.04-nightly
2016 had 690 commits: https://github.com/koreader/koreader/compare/v2015.12.13-nightly...v2016.11.20-nightly
2015 had 419 commits: https://github.com/koreader/koreader/compare/v2014.12.29-nightly...v2015.12.13-nightly

This is all annual statistics I could get :D

**Reply by Frenzie** [reply]
There are many metrics, all with varying degrees of usefulness and uselessness. 👍

**Comment by jonnyl2** [comment]
Thank you. I, too, want to wish everyone Happy Holidays and a fulfilling 2024.

The passion and care that goes into its continued development, and the attentive support shown by the developers, is what makes KOReader such a gem. It's truly one of a kind!
